src/commands/build/addon.rs

```diff
@@ -46,7 +46,7 @@ impl Addon {
 
     pub fn target(&self, p: &Project) -> PathBuf {
         let mut target = PathBuf::from(self.location.to_string());
-        if !p.prefix.is_empty() {
+        if !p.prefix.is_empty() && self.name != p.prefix {
```

Comment:
Not a fan of these checks, the prefix should either be defined or not set

Reply:
should we maybe then add a PathPrefix and a Prefix? because we in some cases need a prefix for the file-patching command

Reply:
File patching should not require a prefix, it only needs the `mainprefix` and the `modname`

Reply:
file patching was current build around the prefix that was the issue I have run into.